src/libraries/Microsoft.PowerFx.Interpreter/DebugDump.cs

```diff
@@ -58,13 +55,15 @@ public static void Dump(ReadOnlySymbolTable symbolTable, TextWriter tw, string i
                 if (symbolTable.Functions.Any())
                 {
                     tw.WriteLine();
-                    tw.WriteLine($"{indent} Functions ({symbolTable.Functions.Count()}) total:"); 
-                    foreach (var func in symbolTable.Functions)
+                    tw.WriteLine($"{indent} Functions ({symbolTable.Functions.Count()}) total:");
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
per other comment - there should be some non-obsolete way to still enumerate all functions. 
Maybe a more explicit method.

Reply:
I added a functionName getter